index.html

```diff
@@ -1 +1,146 @@
+                        <p>Connect with the best experts, in real life, at our summit.
+                         Be part of this unique experience and listen to their
+                          life-changing advice in person!
+                        </p>
+                    </div>
+                    <div class="date-venue">
+                        <p class="date">2022.10.15(THU) ~ 16(FRI)</p>
+                        <p>@ National Museum of Dubai, Art Center Dubai and more</p>
+                    </div>
+                </div>
+            </section>
+            <section class="programSection" id="Program">
+                <div class="program">
+                    <h2>Main Program</h2>
+                    <div class="line"></div>
+                    <div class="programContainer">
+                        <div class="programItem">
+                            <img src="./Assets/makeup.svg" alt="lecture">
+                            <h3>Make-up</h3>
+                            <p>Handpicked from the most reputable brands to provide you the best</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/skincare.svg" alt="exhibition">
+                            <h3>Skin care</h3>
+                            <p>More than just skin deep, Vegan and cruelty-free</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/haircare.svg" alt="forum">
+                            <h3>Hair care</h3>
+                            <p>Say goodbye to bad chemicals and unhealthy ingredients.</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/workshop.svg" alt="workshop">
+                            <h3>Workshop</h3>
+                            <p>More panels and workshops than last year</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/edit-regular.svg" alt="ignite">
+                            <h3>Q & A</h3>
+                            <p>This is your chance to be part of the conversation.</p>
+                            
+                        </div>
+                    </div>
+                    <button class="joinButton mobile"><a href="#">Join Beauty Summit 2022</a></button>
+                    <a href="#" target="_blank" rel="noopener" class="joinLink desktop">See the whole program</a>
+                </div>
+            </section>
+            <section class="speakerSection" id="speakers">
+                <h2 class="speakerTitle">Featured Speakers</h2>
+                <div class="lineS"></div>  
+                <div id="jsSection"></div>
+                <button class="moreButton mobile">More<i class="fas fa-chevron-down icon"></i></button>
+            </section>
```

Comment:
The speaker section of the desktop version should be organized in two columns of three articles each. Currently your desktop version below screen width 1245px displays only single columns. Kindly check and correct this. :pray: 

Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-20 13-06-02](https://user-images.githubusercontent.com/65233860/196957504-ddd54e60-c52d-4de7-bb04-0dde7b0af128.png)  |  ![Screenshot from 2022-10-20 13-06-23](https://user-images.githubusercontent.com/65233860/196957526-bd681c92-9d62-4947-a7dd-fefd6907f8b2.png)


Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-20 13-10-50](https://user-images.githubusercontent.com/65233860/196957746-35d09979-d43e-4eb8-a481-43f0d1d520ce.png)  |  ![Screenshot from 2022-10-20 13-06-23](https://user-images.githubusercontent.com/65233860/196957783-5b3d9ef8-e3b3-4717-b9f7-d5f8cf7c6fc0.png)